Snippets/Core/Core_6/Msmq/DLQOptInForTTBR.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Core6.Msmq
+{
+    using NServiceBus;
+
+    class DLQOptInForTTBR
+    {
+        DLQOptInForTTBR(EndpointConfiguration endpointConfiguration)
+        {
+            #region msmq-dlq-for-ttbr-optin
+
+            endpointConfiguration.UseTransport<MsmqTransport>()
+                .UseDeadLetterQueueForMessagesWithTimeToReachQueue();
```

Comment:
The API says .TimeToReachQueue(), but isn't it TTBR? Since this is MSMQ specific, wouldn't it be better to stick to MSMQ terms?